Headers/DebugServer2/Host/Windows/FileIO.h

```diff
@@ -25,13 +25,13 @@ namespace Windows {
 
 class FileIO {
 public:
-  static ErrorCode OpenFile(std::string const &path, uint32_t flags,
-                            uint32_t mode, int &fd);
-  static ErrorCode CloseFile(int fd);
+  ErrorCode OpenFile(std::string const &path, uint32_t flags, uint32_t mode,
```

Comment:
WTF, I meant RAI, not RTTI :o

On Tue, May 31, 2016 at 3:19 PM, Francis Ricci <notifications@github.com>
wrote:

> In Headers/DebugServer2/Host/Windows/FileIO.h
> <https://github.com/facebook/ds2/pull/353#discussion_r65273462>:
>
> > @@ -25,13 +25,13 @@ namespace Windows {
> >
> >  class FileIO {
> >  public:
> > -  static ErrorCode OpenFile(std::string const &path, uint32_t flags,
> > -                            uint32_t mode, int &fd);
> > -  static ErrorCode CloseFile(int fd);
> > +  ErrorCode OpenFile(std::string const &path, uint32_t flags, uint32_t mode,
>
> By RTTI semantics, do you mean calling these virtual?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/ds2/pull/353/files/4e44056a3d3facb901289ae864a9e8bb629837dd..5e3cd77ac1ebcd965ff84fafe9f0b73b2b564021#r65273462>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADHbJU9qR0oVcsLqeD5dABx6jP_87_yks5qHLPhgaJpZM4ImqRQ>
> .
>



-- 
Stephane Sezer